README.md

```diff
@@ -36,8 +36,10 @@ and a set of options.
 There are three options that must be specified in the [`GridOptions`] value that
 dictate how the grid is formatted:
 
-- [`filling`][filling]: what to put in between two columns — either a number of
-  spaces, or a text string;
+- [`filling`][filling]: how to fill empty space between columns:
+  - [`Filling::Spaces`][Spaces] number of spaces between columns;
```

Comment:
We could make the tabs option a part of the spaces option maybe? What do you think? Something like
```rust
enum Filling {
    Whitespace {
        spaces: usize,
        tab_size: Option<usize>,
    }
    Text(/* ... */),
}
```
What you got works pretty well too, but maybe it would be nice to also configure the separator size for a tab layout. I'd accept this too though!

Reply:
I was trying not to break the API, but introduce a new thing. Maybe we can introduce separator size in new `Tab` instead?

Reply:
That's fair. I like your suggestion!